be/CMakeLists.txt

```diff
@@ -18,13 +18,6 @@
 cmake_minimum_required(VERSION 3.19.2)
 
 # set CMAKE_C_COMPILER, this must set before project command
-if (DEFINED ENV{DORIS_GCC_HOME})
```

Comment:
I think this should be an optional choice?

Reply:
No, We should remove GCC HOME env. 
User should only define CC and CXX env, 
for example
export CC= XXXXXXXXXX
export CXX=xxxxxxxxxxxxxx

I will remove GCC HOME and rewrite bitshuffle build method.

Reply:
If we want to build with clang, then we export CC=XXXX/clang  and CXX=XXXXXXX/clang++
If we want to build with gcc, then we export CC=xxxx/gcc and CXX=xxxxx/g++